src/jwt_util.rs

```diff
@@ -43,10 +43,14 @@ async fn get_rs256_key(client: &Client) -> EsiResult<String> {
 }
 
 /// Decode and validate the JWT token
-fn validate(token: &str, client_id: &str, decoding_key: &DecodingKey) -> Result<TokenClaims, EsiError> {
+fn validate(token: &str, client_id: Option<&str>, decoding_key: &DecodingKey) -> Result<TokenClaims, EsiError> {
```

Comment:
I think this change might've snuck in from last time?

Reply:
Yeah. I think I'm doing something wrong with my pull request, but I'm not sur what. I've corrected the code again.